frontend/src/components/pages/Scheduling/BackButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { Box, Button } from "@chakra-ui/react";
+import { BackButtonProps } from "./types";
+
+export default function BackButton({isBeingEdited, onSaveClick, previous}: BackButtonProps) {
+    return (
+      <Box mt={10}>
+        {isBeingEdited ? (
+          <Button
+            onClick={onSaveClick}
+            variant="navigation"
+          >
+          Save Changes
+        </Button>
+        ) : (
+        <Button
+          onClick={previous}
+          paddingLeft="0 !important"
```

Comment:
let's try to avoid `!important`, since this css property is passed in line, it'll already override and be the highest priority. Usually we use `!important` like you did for custom library stylings to override but should [definitely avoid ](https://uxengineer.com/css-specificity-avoid-important-css/)otherwise! 
```suggestion
          paddingLeft="0"
```